docs/features/tcp-health-probe.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "TCP Health probe"
+layout: default
+---
+
+# TCP Health probe
+
+A `BackgroundService` can be added to the <span>ASP.NET</span> hosted services to expose a TCP health endpoint that allows a runtime to periodically check the liveness/readiness of the host.
```

Comment:
I'm not sure if we have to go so technical, isn't it just enough to say that we provide a TCP health probe endpoint which will expose health information based on .NET Core health approach?

_PS: It's not ASP.NET here so would leave that out maybe?_

Reply:
Ah, right; I'll change this.